c7n/resources/apigw.py

```diff
@@ -1145,3 +1146,86 @@ def process(self, resources, event=None):
             except ClientError as e:
                 if e.response['Error']['Code'] in retryable:
                     continue
+
+
+class WebSocketDescribeSource(query.DescribeSource):
+
+    def augment(self, resources):
+        for r in resources:
+            if 'Tags' not in r:
+                continue
+            r['Tags'] = [{'Key': k, 'Value': v} for k, v in r.pop('Tags', {}).items()]
+        return resources
+
+
+@resources.register('websocket-api')
```

Comment:
not sure that we should really designate this as websocket-api as its also http

Reply:
perhaps apigwv2-api? .. @darrendao @ajkerrigan  @thisisshi  thoughts?

Reply:
API Gateway has three API types: HTTP API, REST API and WebSocket API. I think websocket-api matches aws api gateway definition.

Reply:
api gateway has two different api versions v1 which is rest api, and v2 which has http and web socket, the differentiation on the later is based on attribute (ProtocolType), ie this will pick up http apis, so calling it web socket is a misnomer.

Reply:
Yeah, I changed the resource name to apigatewayv2-api.